coderd/workspaceagents.go

```diff
@@ -1804,6 +1804,9 @@ func convertWorkspaceAgentMetadata(db []database.WorkspaceAgentMetadatum) []code
 	// An empty array is easier for clients to handle than a null.
 	result := []codersdk.WorkspaceAgentMetadata{}
 	for _, datum := range db {
+		if datum.CollectedAt.UTC().IsZero() {
+			datum.CollectedAt = datum.CollectedAt.UTC()
+		}
```

Comment:
Shouldn't we just do `datum.CollectedAt = datum.CollectedAt.UTC()`?

You could also consider using `dbtime.Time(datum.CollectedAt)`, but I realize that function may actually need a `.UTC()` in the returned time.

Reply:
That's what I did initially but wasn't sure if that would potentially cause surprise to consumers of that API. But if we're using `dbtime` for everything we store in postgres then it would probably make this more consistent.

Reply:
I think always returning UTC from the API is sensible, tbh. It'd feel a bit weird if we had the API sometimes return UTC and other times something else. But I suppose this is something we haven't really considered before, and I doubt our API is consistent when PG is in a different TZ.

In fact, we should probably just have middleware that transforms all Go times to UTC when queried.

It's kind of funny we care about UTC when storing the time (doesn't matter when using `timestamptz`) but not when pulling them out (when it actually may matter). 😅

Reply:
Oh, just realized we should just fix all db connections with a `SET TIME ZONE TO 'UTC'`, to make it right for the session (unless that happens already).

Reply:
I could imagine that being surprising to operators, and one could argue that it just masks the underlying issue.